lib/patches/active_record_session_store_patch.rb

```diff
@@ -0,0 +1,9 @@
+module ActiveRecordSessionStorePatch
```

Comment:
Do these belong in lib/patches or lib/extensions?  I think of the former where we are modifying behavior of a class whereas the latter where we are just added behavior to the class.  In these cases, I think this is closer to adding behavior, so I think lib/extensions is more appropriate.

Reply:
That is fair, I can move them.

Reply:
either way, could we please put the dalli patch and the ar session store extension in the same folder?
nicer when the same changes are in the same spot

Reply:
@kbrock are you suggesting they currently aren't?  Or just that when I do move this I should do the same treat mean to the dalli patch (because that was the plan and I think Jason's intent from the comment as well).

Reply:
thanks for pointing out __these__ - just voicing that I'd like to keep them together but looks like you're all set